torch/functional.py

```diff
@@ -167,6 +168,28 @@ def isnan(tensor):
     return tensor != tensor
 
 
+def isinf(tensor):
+    r"""Returns a new tensor with boolean elements representing if each element is `+/-INF` or not.
+
+    Arguments:
+        tensor (Tensor): A tensor to check
+
+    Returns:
+        Tensor: A ``torch.ByteTensor`` containing a 1 at each location of `+/-INF` elements.
+
+    Example::
+
+        >>> torch.isinf(torch.Tensor([1, float('inf'), 2]))
+         0
```

Comment:
This is old style print. Can you update to the new one?

Reply:
updated